download_and_create_reference_datasets/v02/hail_scripts/write_combined_reference_data_ht.py

```diff
@@ -294,20 +362,20 @@ def join_hts(datasets, coverage_datasets=[], reference_genome='37'):
                                          datasets=hl.dict(included_dataset),
                                          version=VERSION)
     joined_ht.describe()
-
-    output_path = os.path.join(OUTPUT_TEMPLATE.format(genome_version=reference_genome, version=VERSION))
-    print('Writing to %s' % output_path)
-
-    joined_ht.write(os.path.join(output_path))
+    return joined_ht
 
 
 def run(args):
-    hl._set_flags(no_whole_stage_codegen='1') # hail 0.2.78 hits an error on the join, this flag gets around it 
-    join_hts(['cadd', '1kg', 'mpc', 'eigen', 'dbnsfp', 'topmed', 'primate_ai', 'splice_ai', 'exac',
+    # hl._set_flags(no_whole_stage_codegen='1') # hail 0.2.78 hits an error on the join, this flag gets around it
```

Comment:
this should either stay the same or be fully removed, but we don't want to just keep it around commented out

Reply:
Remove it.